extension/docs/HowToRun.md

```diff
@@ -18,21 +18,61 @@
+
+```json
+{
+  "commercetools": {
+    "commercetoolsProjectKey1": {
+      "clientId": "xxx1",
+      "clientSecret": "xxx1",
+      "ensureResources": true
+    }
+  },
+  "adyen": {
+    "adyenMerchantAccount1": {
+      "apiKey": "xxx2",
+      "clientKey": "xxx2"
+    }
+  },
+  "logLevel": "DEBUG",
+  "port": 8080,
+  "apiExtensionBaseUrl": "url-where-the-extension-module-is-deployed",
+  "keepAliveTimeout": 10000
+}
+```
 
-Following environment variables must be provided in order to run the extension module.
+The JSON structure will be described in details in the next sections of this documentation.
 
 ### Adyen
 
 - For **test environment** follow the official Adyen [get started guide](https://docs.adyen.com/checkout/get-started) to set up your **test account**, get your API key.
 - For **live environment** follow the official Adyen [documentation](https://docs.adyen.com/user-management/get-started-with-adyen#step-2-apply-for-your-live-account) for details.
 
-| Name                        | Content                                                                                                                                                  | Required | Default value (only for test environment)            |
-| --------------------------- | -------------------------------------------------------------------------------------------------------------------------------------------------------- | -------- | ---------------------------------------------------- |
-| `ADYEN_MERCHANT_ACCOUNT`    | The name of your merchant account.                                                                                                                       | YES      |                                                      |
-| `ADYEN_API_KEY`             | You'll be making API requests that are authenticated with an [API key](https://docs.adyen.com/user-management/how-to-get-the-api-key#page-introduction). | YES      |                                                      |
-| `ADYEN_API_BASE_URL`        | [Checkout endpoint](https://docs.adyen.com/development-resources/live-endpoints#checkout-endpoints) of Adyen.                                            | YES      | `https://checkout-test.adyen.com/v52`                |
-| `ADYEN_LEGACY_API_BASE_URL` | [Standart payment endpoint](https://docs.adyen.com/development-resources/live-endpoints#standard-payments-endpoints) of Adyen.                           | YES      | `https://pal-test.adyen.com/pal/servlet/Payment/v52` |
+Multiple child attributes can be provided in the `adyen` attribute. Each direct child attribute must represent 1 adyen merchant account like in the following example:
+
+```
+{
+  "adyen": {
+    "adyenMerchantAccount1": {  // The name of your first merchant account.
+      "apiKey": "xxx"
+      "apiBaseUrl": "https://checkout-test.adyen.com/v50",
```

Comment:
Actually I don't want to use default, because if you use default, then you don't overwrite them :) You specify `apiBaseUrl` only when you don't want to use default.